src/pages/TodoPage/TodoPageContainer.tsx

```diff
@@ -21,14 +32,36 @@ export const TodoPageContainer: React.FC = () => {
     setVisibleModal((prev) => !prev);
   };
 
+  const handleTodoProgress = (id: string, progress: string) => {
+    setTodos(
+      todos.map((todo) =>
+        todo.id === id ? { ...todo, progress: progress } : todo
+      )
+    );
+  };
+
+  const filterTodos = (title: string) => {
+    if (title === 'All') {
+      setCurrentFilter(title);
+      setFilteredTodos(todos);
```

Comment:
add 'return' to avoid else construction
`setCurrentFilter(title);` move out from the condition expression